docs/tutorial/part-four/index.md

```diff
@@ -200,21 +181,21 @@ module.exports = {
 }
 ```
 
-Add the above files and then run `gatsby develop`, per usual, and you should see the following:
+उपयुर्क्त फ़ाइलों को ऐड करें और फिर सामान्य रूप से `gatsby develop` को चलाएं, और आपको निम्नलिखित दिखेगा:
 
 ![start](start.png)
 
-You have another small site with a layout and two pages.
+आपके पास एक और छोटी साइट है जिसमें एक लेआउट और दो पेजेज हैं।
 
-Now you can start querying 😋
+अब आप quering करना शुरू कर सकते हैं 😋
 
-## Your first GraphQL query
+## आपका पहला GraphQL query
```

Comment:
`आपका पहला GraphQL query` => `आपकी पहला GraphQL query`

Reply:
shouldn't it be ```आपकी पहली GraphQL query```